### DISCUSSION TITLE
some problems in python

### DISCUSSION THREAD

**Discussion by DeokjinLee** [raisimTech/raisimLib#233] [Q&A]
Hello, Prof.Hwangbo.

Thanks for providing an amazing simulator.
There are 2 problems using python.

1. Getting end-effector position
I want to use getFramePosition(~, ee_pos) function but got an error due to 'ee_pos' (I guess).
In c++, ee_pos should be defined as raisim::Vec<3> ee_pos. Then, in python, how should it be defined?
I tried ee_pos = np.zeros(3) / ee_pos = raisim.vec<3> etc ... but all didn't work. ​

2. Importing xacro file
I tried  importing xacro file as below but it didn't work.
Robot_xacro_file = '~/rsc/iiwa/iiwa14.urdf.xacro'
robot = world.addArticulatedSystem(robot_xacro_file)
Can you provide code for this?

3. Raisim python example
I've referring to Brian Delhaisse' github (co-developer of raisim).
If there are more resources, please let me know.

Thank you.

**Comment by jhwangbo** [comment]
1.  it should directly output the position. The error message should tell you what you should do
2.  Xacro is not supported

**Reply by DeokjinLee** [reply]
Solved the problems. 
Thank you for your help.
